test/unit/clarity_ext/test_units.py

```diff
@@ -0,0 +1,25 @@
+import unittest
+from clarity_ext.units import Units
+
```

Comment:
Good point. I've renamed it to `UnitConversion` and am now importing it on the root package level, so it can be used with `from clarity_ext import UnitConversion`.